data/sql/updates/pending_db_world/rev_1675639737114759100.sql

```diff
@@ -0,0 +1,232 @@
+(3, 0, 21594),
+(3, 0, 14387),
+(3, 0, 50381),
+(3, 0, 15031),
+(3, 0, 26987),
+(3, 0, 29241),
+(3, 0, 32333),
+(3, 0, 33115),
+(3, 0, 37085),
+(3, 0, 41759),
+(3, 0, 84459),
+(3, 0, 78685),
+(3, 0, 62090),
+(3, 0, 72388),
+(3, 0, 86832),
+(3, 0, 67040),
+(3, 0, 78781),
+(3, 0, 65108),
+(3, 0, 63688),
+(3, 0, 59383),
+(3, 0, 63625),
+(3, 0, 70021),
+(3, 0, 82071),
+(3, 0, 117903),
+(3, 0, 111075),
+(3, 0, 101136),
+(3, 0, 101303),
+(3, 0, 122686),
+(3, 0, 117065),
+(3, 0, 202337),
+(3, 0, 2017),
+(3, 0, 132683);
+
+-- remove potential duplicates
+DELETE FROM `spawn_group` WHERE `groupId` != 3 AND `spawnType`=0 AND `spawnId` IN (SELECT `spawnId` FROM (SELECT `spawnId` FROM `spawn_group` WHERE `groupId`=3  AND `spawnType`=0) as `temp`);
+DELETE FROM `spawn_group` WHERE `groupId` != 4 AND `spawnType`=1 AND `spawnId` IN (SELECT `spawnId` FROM (SELECT `spawnId` FROM `spawn_group` WHERE `groupId`=4  AND `spawnType`=1) as `temp`);
+
+-- Update Acore strings for various cs_list strings, to support showing spawn ID and guid.
+UPDATE `acore_string`
+SET `content_default` = '%d (Entry: %d) - |cffffffff|Hgameobject:%d|h[%s X:%f Y:%f Z:%f MapId:%d]|h|r %s %s'
+WHERE `entry` = 517;
+
+UPDATE `acore_string`
+SET `content_default` = '%d - |cffffffff|Hcreature:%d|h[%s X:%f Y:%f Z:%f MapId:%d]|h|r %s %s'
+WHERE `entry` = 515;
+
+UPDATE `acore_string`
+SET `content_default` = '%d - %s X:%f Y:%f Z:%f MapId:%d %s %s'
+WHERE `entry` = 1111;
+
+UPDATE `acore_string`
+SET `content_default` = '%d - %s X:%f Y:%f Z:%f MapId:%d %s %s'
+WHERE `entry` = 1110;
+
+-- Add new Acore strings for extra npc/gobject info lines
+DELETE FROM `acore_string` WHERE `entry` BETWEEN 5070 AND 5082;
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES
+(5070, 'Spawn group: %s (ID: %u, Flags: %u, Active: %u)'),
+(5071, 'Compatibility Mode: %u'),
+(5072, 'GUID: %s'),
+(5073, 'SpawnID: %u, location (%f, %f, %f)'),
+(5074, 'Distance from player %f'),
+(5075, 'Spawn group %u not found'),
+(5076, 'Spawned a total of %zu objects:'),
+(5077, 'Listing %s respawns within %uyd'),
+(5078, 'Listing %s respawns for %s (zone %u)'),
+(5079, 'SpawnID | Entry | GridXY| Zone | Respawn time (Full)'),
+(5080, 'overdue'),
+(5081, 'creatures'),
+(5082, 'gameobjects');
+
+-- Add new NPC/Gameobject commands
+DELETE FROM `command` WHERE `name` IN ('npc spawngroup', 'npc despawngroup', 'gobject spawngroup', 'gobject despawngroup', 'list respawns');
+INSERT INTO `command` (`name`, `security`, `help`) VALUES
+('npc spawngroup', 3, 'Syntax: .npc spawngroup $groupId [ignorerespawn] [force]'),
+('npc despawngroup', 3, 'Syntax: .npc despawngroup $groupId [removerespawntime]'),
+('gobject spawngroup', 3, 'Syntax: .gobject spawngroup $groupId [ignorerespawn] [force]'),
+('gobject despawngroup', 3, 'Syntax: .gobject despawngroup $groupId [removerespawntime]'),
+('list respawns', 3, 'Syntax: .list respawns [distance] Lists all pending respawns within <distance> yards, or within current zone if not specified.');--
```

Comment:
```suggestion
('list respawns', 3, 'Syntax: .list respawns [distance] Lists all pending respawns within <distance> yards, or within current zone if not specified.');
```
This makes SQL CI madge